lib/main.coffee

```diff
@@ -0,0 +1,25 @@
+
+module.exports =
+  provideLinter: ->
+    jshintPath = require('path').join(__dirname, '..', 'node_modules', '.bin', 'jshint')
```

Comment:
I would love to have the support of locally `jshint` binary installed in our project, example:

* I'm working on open source project, they use `x.y.z` version of jshint which is declared in dev deps of package.json
* On my own private project I use the jshint installed on my machine

This is common use case, you can use something like: https://www.npmjs.com/package/npm-which to achieve it.

btw, thank's for the example going to update `linter-jscs` and `linter-eslint` with the new CLI helpers tomorrow.